File: sorting/bubbleSort.js
function bubbleSort(customList) {
  for (let i = 0; i < customList.length - 1; i++) {
    for (let j = 0; j < customList.length - i - 1; j++) {
      if (customList[j] > customList[j + 1]) {
        let temp = customList[j];
        customList[j] = customList[j + 1];
        customList[j + 1] = temp;
      }
    }
  }
  console.log(customList);
}

let list1 = [2, 1, 7, 6, 5, 3, 4, 8];
let list2 = [2, 1, 8];

bubbleSort(list1);
bubbleSort(list2);

File: sorting/bubble_sort.py
def bubble_sort(customList):
    for i in range(len(customList) - 1):
        for j in range(len(customList) - i - 1):  # decrease number of loops as
            if customList[j] > customList[j + 1]:
                customList[j], customList[j +
                                          1] = customList[j + 1], customList[j]
    print(customList)


# compare adjacent elements
# note how move from pair to pair
# numbers go to right edge (bubble up)

list1 = [2, 1, 7, 6, 5, 3, 4, 8]
bubble_sort(list1)
